libraries/AP_Scripting/generator/Makefile

```diff
@@ -13,7 +13,7 @@ OBJS = $(patsubst %,$(OBJ_DIR)/%,$(_OBJS))
 
 .PHONY: all
 all: directories $(OBJS)
-	$(CC) $(LDFLAGS) $(OBJS) -o $(OUT)
+	$(CC) $(OBJS) -o $(OUT) $(LDFLAGS)
```

Comment:
Does this change any behavior?

Reply:
@magicrub san.   I got an error in the build. It was able to build normally if it changed.
====
This change:
muramura@Aspire:~/work/ardupilot/libraries/AP_Scripting/generator$ make
mkdir -p build
mkdir -p build/obj
gcc -Wall -Werror -Wextra -g -c src/main.c -o build/obj/main.c.o
gcc build/obj/main.c.o -o build/gen-bindings -lreadline
cp build/gen-bindings .

====
before:
muramura@Aspire:~/work/ardupilot/libraries/AP_Scripting/generator$ make
mkdir -p build
mkdir -p build/obj
gcc -Wall -Werror -Wextra -g -c src/main.c -o build/obj/main.c.o
gcc -lreadline build/obj/main.c.o -o build/gen-bindings 
build/obj/main.c.o: In function `start_line':
/home/muramura/work/ardupilot/libraries/AP_Scripting/generator/src/main.c:198: undefined reference to `readline'
collect2: error: ld returned 1 exit status
Makefile:16: recipe for target 'all' failed
make: *** [all] Error 1